docker-compose.env.sample

```diff
@@ -19,3 +19,4 @@ NOTIFY_SUBSCRIPTION_CONFIRMATION_TEMPLATE=
 SUBSCRIPTION_KEY_GENERATOR_SECRET=10dc5b4d246466a71aeb9fd87a41207b6ec77a38096cf3f5a0948981f5c873a9f7cd1c3444cfc7cdc8e2b37dbabfc31e0c35a597f552e683eb471555e8c75834
 SUBSCRIPTION_KEY_GENERATOR_SALT=fc811c17d10be2aa5467237eef8cb8c8f6edb493fcc120c1b9e2db67159cce073398d660167370c24d5eadab56ee93ba15f72e3824ada843e43232ec31640393
 FEATURE_EMAIL_ALERTS=true
+AUDIT_SUBSCRIPTION_CREATION_GID=
```

Comment:
Small request: can we put this next to the spreadsheet ID, so it's easier to parse what this refers to?